packages/flutter_tools/tool/run_local_linter.dart

```diff
@@ -0,0 +1,177 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Was this just used to test the change to the linter, or does this actually need to land? Wouldn't the new lint just be enabled in the tool's analysis options yaml?

Reply:
You are right, once the lint rolls it will be enabled via analysis_options.yaml. My thinking with landing this was that it would be helpful in future debugging/development of upstream lints. I'm also thinking this script could be extended to lint dependency code (which the analyzer doesn't support).

However, I can just as easily keep this in a personal scripts repo. WDYT?

Reply:
If there's some way to make it run in CI, then I'm okay with keeping it if it's useful. Just reluctant to have dead code checked in.

Reply:
I think it would be cumbersome (and flaky) to write an integration test for this, so i'll just maintain it separately.